docs/source/hackrf_tools.rst

```diff
@@ -0,0 +1,139 @@
+============
+HackRF Tools
+============
+
+Great Scott Gadgets provides some commandline tools for interacting with HackRF. 
+    * **hackrf_info** Read device information from HackRF such as serial number and firmware version.
+
+    * **hackrf_transfer** Send and receive signals using HackRF. Input/output files are 8-bit signed quadrature samples.
+
+    * **hackrf_sweep**, a command-line spectrum analyzer.
+
+    * **hackrf_clock** Read and write clock input and output configuration.
+
+    * **hackrf_operacake** Configure Opera Cake antenna switch connected to HackRF.
+
+    * **hackrf_spiflash** A tool to write new firmware to HackRF. See: :ref:`Updating Firmware <updating_firmware>`.
+
+    * **hackrf_debug** Read and write registers and other low-level configuration for debugging.
+
+
+
+hackrf_sweep
+~~~~~~~~~~~~
+
+Usage
+^^^^^
+
+.. code-block:: sh 
+
+    [-h] # this help
+    [-d serial_number] # Serial number of desired HackRF
+    [-a amp_enable] # RX RF amplifier 1=Enable, 0=Disable
+    [-f freq_min:freq_max] # minimum and maximum frequencies in MHz
+    [-p antenna_enable] # Antenna port power, 1=Enable, 0=Disable
+    [-l gain_db] # RX LNA (IF) gain, 0-40dB, 8dB steps
+    [-g gain_db] # RX VGA (baseband) gain, 0-62dB, 2dB steps
+    [-w bin_width] # FFT bin width (frequency resolution) in Hz, 2445-5000000
+    [-1] # one shot mode
+    [-N num_sweeps] # Number of sweeps to perform
+    [-B] # binary output
+    [-I] # binary inverse FFT output
+    -r filename # output file
+
+
+
+Output fields
+^^^^^^^^^^^^^
+
+``date, time, hz_low, hz_high, hz_bin_width, num_samples, dB, dB, ...``
```

Comment:
I don't understand what you mean by "runs off the edge". It looks like the entire table is rendered correctly to me.

Reply:
The table is correct; it just isn't responsive. For comparison, this is what that table looks like in https://hackrf.readthedocs.io/en/latest/hackrf_sweep.html:
![Screen Shot 2023-04-07 at 4 39 27 PM](https://user-images.githubusercontent.com/19697276/230687602-42904f69-7c25-49f0-bc4d-cbefea6c23d4.png)

Reply:
The issue you are experiencing still isn't clear to me, but I will try my best to interpret.

I am assuming that what you mean by 'responsive' is that you would like the table to have a horizontal scroll bar. In the first image you posted of the table in this thread, the table is displaying all of the information expected. If all the information in a table is displayed on your screen a horizontal scroll bar will not be generated by the web framework. 

In the second picture of the table you posted, it looks like you are looking at the table on a smaller screen, or the window is smaller, so the table is not displaying all of the information at once. Since all of the information is not displayed at once the web framework has added a horizontal scroll bar. 

It still isn't clear to me what you mean by 'runs off the edge' in your comment https://github.com/greatscottgadgets/hackrf/pull/1295#discussion_r1156193628. I assume that you thought the table had more information to display when it actually didn't have more information to display. The table ends with five dB columns.

Reply:
I'm sorry it's not clear. I think it's because the screenshots I shared aren't showing a 1:1 view of the page comparison. To clarify-yes, I am expecting to see a horizontal scroll bar. 

Here's what I expect to see: 

<img width="1424" alt="Screen Shot 2023-04-08 at 8 19 00 AM" src="https://user-images.githubusercontent.com/19697276/230726774-288961a5-707e-43ad-b4a1-b1f1d2dd4574.png">

Here's what I see when I preview the HTML of the hackrf_tools page on your branch:

<img width="1430" alt="Screen Shot 2023-04-08 at 8 19 23 AM" src="https://user-images.githubusercontent.com/19697276/230726796-813e6bc1-0938-493f-89e7-7236abf89775.png">

Reply:
These are screenshots are of different tabs on the same browser window.

Reply:
Thank you for the pictures. Turns out that when displayed on my monitor I can't see the table going off of the edge of the white background; the light grey shows up as white. When viewed on my phone it is much more obvious what you are referring to. Thank you for your patience in helping resolve this issue. I'm working on a fix now.

Reply:
I have built the documentation on my computer and opened the HackRF Tools file in both FireFox and Chrome and the table has a horizontal scroll bar in both instances. Which browser and OS combination are you testing your documentation build on?

Reply:
I have sent you a copy of my build of the docs to see if when you open them the table issue still exists.

Reply:
> I have built the documentation on my computer and opened the HackRF Tools file in both FireFox and Chrome and the table has a horizontal scroll bar in both instances. Which browser are you testing your documentation build on? What version of Sphinx docs are you using?

I am testing the documentation build on Chrome and using Sphinx docs version 6.1.3.

Reply:
> I have sent you a copy of my build of the docs to see if when you open them the table issue still exists.
The issue does not exist with the build you sent me!

Reply:
Great! Thanks.